src/components/Layout/Layout.tsx

```diff
@@ -15,9 +16,19 @@ const content = css({
 });
 
 const Layout = () => {
+  const location = useLocation();
+  const [hideHeader, setHideHeader] = useState<boolean>(false);
+
+  useEffect(() => {
+    if (location.pathname === '/') {
+      setHideHeader(true);
+    }
+  }, [location]);
```

Comment:
이거 굳이 상태일 필요가 없을 것 같아요!
```suggestion
const isHomePage = location.pathname === '/';

{!isHomePage && <header/>}
```

이런 식으로 해주시면 상태와 useEffect 없어도 될 것 같습니다

Reply:
수정했습니다